msal/tests/dev apps/XForms/XForms/RefreshTokenCacheItemDetails.xaml.cs

```diff
@@ -39,21 +39,22 @@ namespace XForms
     [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class RefreshTokenCacheItemDetails : ContentPage
     {
-        internal RefreshTokenCacheItemDetails(MsalRefreshTokenCacheItem msalRefreshTokenCacheItem)
+        internal RefreshTokenCacheItemDetails(MsalRefreshTokenCacheItem msalRefreshTokenCacheItem, 
+            MsalAccountCacheItem MsalAccountCacheItem)
```

Comment:
>M [](start = 33, length = 1)

lower case ? it seems like a static this way or a property.

Reply:
will change